addon/-private/object.js

```diff
@@ -50,6 +50,17 @@ const proxyHandler = {
     return true;
   },
 
+  deleteProperty(target, prop, receiver) {
```

Comment:
Hmm, according to the spec, `deleteProperty` has no `receiver`.

Reply:
True if you look at an Object’s `deleteProperty`, but in this case, we’re dealing with a [Proyx](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy).

Reply:
I'm sorry, but I don't really understand. What is "Object's `deleteProperty`"?
I meant [proxyHandler.deleteProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/Proxy/deleteProperty) has no `receiver` argument, https://github.com/tc39/ecma262/issues/1198

Reply:
I think you’re right, @nag5000. 👍 I got confused.